proto/dualitylabs/duality/dex/limit_order_tranche.proto

```diff
@@ -4,47 +4,57 @@ package dualitylabs.duality.dex;
 option go_package = "github.com/duality-labs/duality/x/dex/types";
 import "google/protobuf/timestamp.proto";
 
+import "dualitylabs/duality/dex/trade_pair_id.proto";
 import "gogoproto/gogo.proto";
 import "dualitylabs/duality/dex/pair_id.proto";
 
+message LimitOrderTrancheKey {
+  TradePairID tradePairID = 1;
+  int64 tickIndexTakerToMaker = 2;
+  string trancheKey = 3; 
```

Comment:
It's a bit confusing to have LimitOrderTrancheKey and `trancheKey`

Reply:
TrancheKey should be TrancheId but we'll have to fix this forward.